example/blog/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "mongoose": "5.6.0",
+    "mongoose": "5.9.7",
```

Comment:
Mongoose version 6

Reply:
As i already mentioned, the example folder will be updated once this PR is approved.